cmd/sops/common/common.go

```diff
@@ -139,19 +170,40 @@ func IsIniFile(path string) bool {
 	return strings.HasSuffix(path, ".ini")
 }
 
+// DefaultStoreForFormat returns the correct format-specific implementation
+// of the Store interface given the format string
+func DefaultStoreForFormat(format string) Store {
```

Comment:
With my suggested changes, this should not be needed anymore.

Reply:
Either `StoreConstructors` needs to be public or `DefaultStoreForFormat` needs to be public, so the CLI can consume it.

Reply:
Yeah, you're right. I think I'd expose `Format` and the associated way to get a Store from a given format in the `go.mozilla.org/sops` package directly (so at the root of the repository) since it isn't command-line specific. It's not super important since it's not part of the public API, though, so feel free to just keep it here.